README.md

```diff
@@ -105,5 +105,17 @@ Then we can run:
     In [9]: car.driver.married_to.married_to.name
     Out[9]: 'Kowalski'
 
+## Authorization HTTP header forwarding example
+
+We can set jsonapi_requests.auth.FlaskForwardAuth() as AUTH configuration option to copy authorization header from current request context.
```

Comment:
Let's add some mention about Flask. It's in the class name, but well, still worth mentioning that it's a Flask-related magic.